pkg/auth/authproviders/openshift/internal/dexconnector/openshift.go

```diff
@@ -281,11 +281,10 @@ func (c *openshiftConnector) identity(ctx context.Context, s connector.Scopes, t
 	}
 
 	identity = connector.Identity{
-		UserID:            user.UID,
-		Username:          user.Name,
-		PreferredUsername: user.Name,
```

Comment:
Wait what version are we using that doesn't have this field anymore?

[Upstream it is still available](https://github.com/dexidp/dex/blame/e41a28bf27225ab503eb9feef4feedd03bb4ac71/connector/connector.go#L28)?


Ah, I see v2.12.0 doesn't have it...
Given that the commit we've used prior to this ([2bb4896d120e](https://github.com/dexidp/dex/commit/2bb4896d120e)), do you think we can just upgrade to `v2.37.0`?

Reply:
Field removal should be fine but let's ask the author of this code @rukletsov What do you think?

Reply:
Looks like it's fine to remove PreferredUsername, we don't use it apparently at the moment: https://github.com/stackrox/stackrox/blob/e12a15a27395ccec2e594112a9e0c6e71a378f79/pkg/auth/authproviders/openshift/backend_impl.go#L150-L170